### ISSUE TITLE
Loosing Control For Apps and Mic in ubuntu budgie desktop 17.10.1

### ISSUE THREAD

**Issue by ujjwal117** [wwmm/pulseeffects#183]
i have been using pulse-effects for few days and its great app but it taking over control for audio and mic so i have to turn off the mic always if i want to use for skype i cant the sound echoing and making noise. And every time i had to configure the audio setting for playback and it always doesnot solve the problem previously i installed pulse audio equalizer so removed it and then pulse effects does not gave any output over playback audio, hence i reinstalled pulseffects but problem was same, so i think this could be a bug. Thanks for your work. Appreciate it i was searching for limiter and compressor and when finally i found it was not worked correctly.

**Comment by wwmm** [comment]
I didn't understand (English is not my native language). What do you mean by ` turning off the mic` and `taking over control for audio`?

If you have the option `Enable All Applications` on PulseEffects will process the output of applications and process the microphone signal that they are trying to record. Something you also must have in mind is that until the next version of Pulseaudio's is released the module `module-device-manager`  used by kde and the module `module-switch-on-connect` loaded by default in the latest ubuntu are not compatible with PulseEffects and have to be unloaded before starting PE. HAve a look at [FAQ](https://github.com/wwmm/pulseeffects/wiki/FAQ)

**Comment by wwmm** [comment]
closing due to inactivity

**Comment by mikhailnov** [comment]
@ujjwal117 try PulseAudio 12.0 on Ubuntu 18.04, see https://github.com/wwmm/pulseeffects/issues/227#issuecomment-401391688 for instructions
@wwmm it seems to be a duplicate of https://github.com/wwmm/pulseeffects/issues/99

**Comment by wwmm** [comment]
I agree. It is probably the same issue
